web/.eslintrc.json

```diff
@@ -23,9 +23,10 @@
     ],
     "react/prop-types": "off",
     "no-useless-catch": "off",
+    "@typescript-eslint/ban-ts-comment": "warn",
     "@typescript-eslint/camelcase": "off",
     "@typescript-eslint/explicit-function-return-type": "off",
-    "@typescript-eslint/ban-ts-comment": "warn"
+    "@typescript-eslint/no-non-null-assertion": "off"
```

Comment:
We lose a lot of type safety if we turn this off globally. Is there another way or can we scope this down to the specific cases where we need this?

Reply:
It was my laziness that went this route. I will revert this

Reply:
Done